app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,18 @@
  *= require_tree .
  *= require_self
  */
+
+ #share-buttons img {
+ width: 20px;
+ padding: 2px;
+ display: inline;
+ text-decoration: none;
+ }
+
+ a:hover {
+     opacity: 0.7;
+ }
+
+a {
+  color: white;
+}
```

Comment:
Best to have a file called `_share_buttons.scss` — rails will require it.

The indentation is also off in this file, look at lines 25 and 29.